src/EnergyPlus/PackagedTerminalHeatPump.cc

```diff
@@ -4804,8 +4805,8 @@ void InitPTUnit(EnergyPlusData &state,
             if (NoCompOutput > QToCoolSetPt) {
                 QZnReq = QToCoolSetPt;
                 //       Don't set mode TRUE unless mode is allowed. Also check for floating zone.
-                if (state.dataHeatBalFanSys->TempControlType(ZoneNum) == SingleHeatingSetPoint ||
-                    state.dataHeatBalFanSys->TempControlType(ZoneNum) == 0) {
+                if (state.dataHeatBalFanSys->TempControlType(ZoneNum) == DataHVACGlobals::SetPointType::SingleHeating ||
+                    state.dataHeatBalFanSys->TempControlType(ZoneNum) == DataHVACGlobals::SetPointType::Uncontrolled) {
```

Comment:
Same comment. Don't use `if/else` to set things to `true` or `false`. Just set the variable to the `if` condition or its negative.